backend/src/main/java/com/hulk/organicfarm/controllers/FileUploadController.java

```diff
@@ -11,38 +12,42 @@
 import org.springframework.web.bind.annotation.RequestParam;
 import org.springframework.web.bind.annotation.ResponseBody;
 import org.springframework.web.multipart.MultipartFile;
-import org.springframework.web.servlet.mvc.support.RedirectAttributes;
+
+import java.util.HashMap;
+import java.util.Map;
 
 import com.hulk.organicfarm.services.StorageService;
 
 @Controller
```

Comment:
Remove controller and add these 
@CrossOrigin(origins = "*")
@RestController
@RequestMapping("/api")